widgets/hierarchical-menu/hierarchical-menu.js

```diff
@@ -81,7 +81,7 @@ function hierarchicalMenu({
         templates
       });
 
-      cssClasses = {
```

Comment:
I was wondering also, I tested it that's fine: https://babeljs.io/repl/#?experimental=true&evaluate=true&loose=false&spec=false&code=function%20hierarchicalMenu(%7B%0A%20%20%20%20container%2C%0A%20%20%20%20attributes%20%3D%20%5B%5D%2C%0A%20%20%20%20separator%2C%0A%20%20%20%20limit%20%3D%20100%2C%0A%20%20%20%20sortBy%20%3D%20%5B'name%3Aasc'%5D%2C%0A%20%20%20%20cssClasses%3A%20userCssClasses%20%3D%20%7B%7D%2C%0A%20%20%20%20hideContainerWhenNoResults%20%3D%20true%2C%0A%20%20%20%20templates%20%3D%20defaultTemplates%2C%0A%20%20%20%20transformData%0A%20%20%7D)%20%7B%0A%20%20%20%20%0A%7D%3B